src/data/legacy/groups.mdx

```diff
@@ -0,0 +1,215 @@
+---
+title: "Groups"
+description: "Manage user groups."
+---
+
+# List
+
+List all existing user groups.
+
+<Endpoint method="GET" path="/api/groups" />
+
+## Example
+
+<Request method="GET" path="/api/groups" />
+
+## Response
+
+<Response status="200 OK">
+```json
+[
+  {
+    "permissions": {
+      "cancel_job": true,
+      "create_ref": false,
+      "create_sample": true,
+      "modify_hmm": false,
+      "modify_subtraction": false,
+      "remove_file": false,
+      "remove_job": true,
+      "upload_file": true
+    },
+    "id": "technicians"
+  }
+]
+```
+</Response>
+
+## Errors
+
+_None_
+
+# Get
+
+Get the complete representation of a single user group.
+
+<Endpoint method="GET" path="/api/groups/:id" />
+
+## Example
+
+<Request method="GET" path="/api/groups/technicians" />
+
+## Response
+
+<Response status="200 OK">
+
+```json
+{
+  "permissions": {
+    "cancel_job": true,
+    "create_ref": false,
+    "create_sample": true,
+    "modify_hmm": false,
+    "modify_subtraction": false,
+    "remove_file": false,
+    "remove_job": true,
+    "upload_file": true
+  },
+  "id": "technicians"
+}
+```
+
+</Response>
+
+## Errors
+
+| Status | Message   | Reason               |
+| :----- | :-------- | :------------------- |
+| `404`  | Not found | group does not exist |
+
+# Create
+
+<administrator />
+
+Create a new group. New groups have no permissions. Requestors must be administrators.
+
+<Endpoint method="POST" path="/api/groups" />
+
+## Input
+
+| Name     | Type   | Description                                |
+| :------- | :----- | :----------------------------------------- |
+| group_id | string | a unique id and display name for the group |
+
+## Example
+
+<Request method="POST" path="/api/groups">
+
+```json
+{
+  "group_id": "research"
+}
+```
+
+</Request>
+
+## Response
+
+<Response status="201: Created">
+
+```json
+{
+  "permissions": {
+    "cancel_job": false,
+    "create_ref": false,
+    "create_sample": false,
+    "modify_hmm": false,
+    "modify_subtraction": false,
+    "remove_file": false,
+    "remove_job": false,
+    "upload_file": false
+  },
+  "id": "research"
+}
+```
+
+</Response>
+
+## Errors
+
+| Status | Message              | Reason                                 |
+| :----- | :------------------- | :------------------------------------- |
+| `400`  | Group already exists | the provided `group_id` already exists |
+| `403`  | Not permitted        | client is not an administrator         |
+| `422`  | Invalid input        | JSON request body is invalid           |
+
+# Edit
+
+<administrator />
```

Comment:
```suggestion
<Administrator />
```
This is a React component. They always started with an uppercase letter.

There is no `Administrator` component. Please open an issue and assign it to me.